src/java/com/zimbra/oauth/handlers/impl/FacebookOAuth2Handler.java

```diff
@@ -167,6 +173,42 @@
          */
         public static final String HOST_FACEBOOK = "graph.facebook.com";
 
+        /**
+         * The contacts uri template.
+         */
+        public static final String CONTACTS_URI_TEMPLATE =
+            "https://graph.facebook.com/v3.0/me/friends?access_token=%s&fields=%s&limit=%s";
+
+        /**
+        * The contacts pagination size for Facebook.
+        */
+        protected static final String CONTACTS_PAGE_SIZE = "100";
+
+        /**
+         * The refresh token code request uri template.<br>
+         * Uses a code to request a fresh access token.
+         */
+        public static final String REFRESH_TOKEN_CODE_REQUEST_URI_TEMPLATE =
+            "https://graph.facebook.com/oauth/client_code"
+            + "?access_token=%s&client_id=%s&client_secret=%s&redirect_uri=%s";
+
+        /**
+         * The access request uri template code, uses the existing, valid access token
+         * to fetch a code.
+         * This code will be used to request a fresh access token.
+         */
+        public static final String REFRESH_ACCESS_TOKEN_FOR_CODE_REQUEST_URI_TEMPLATE =
+            "https://graph.facebook.com/oauth/access_token"
+            + "?client_id=%s&redirect_uri=%s&code=%s";
+
+        /*
+         * CSV list of data fields to import or limit the import to.
+         * (Please note that this list may require permission scopes
+         * be added to the localconfig.xml for the related fields)
+         */
+        public static final String IMPORT_FIELDS_LIST = "email,address,name,location,"
```

Comment:
This is only used in one place - when formatting the contacts fetch uri. Your import code is only written to handle these properties so just combine the two and remove this extra constant.